website/content/docs/connect/config-entries/sameness-group.mdx

```diff
@@ -302,7 +325,7 @@ The following examples demonstrate common sameness group configuration patterns
 
 ### Failover between members of a sameness group
 
-In the following example, the configuration entry defines a sameness group named `products-api` that applies to the `store-east` partition in the local datacenter. The sameness group is configured so that when its services’ instances in `store-east` fails, Consul will attempt to establish a failover connection in the following order:
+In the following example, the configuration entry defines a sameness group named `products-api` that applies to the `store-east` partition in the local datacenter. The sameness group is configured so that when a service instance in `store-east` fails, Consul will attempt to establish a failover connection in the following order:
```

Comment:
```suggestion
In the following example, the configuration entry defines a sameness group named `products-api` that applies to the `store-east` partition in the local datacenter. The sameness group is configured so that when a service instance in `store-east` fails, Consul attempts to establish a failover connection in the following order:
```